nvflare/private/fed/server/server_commands.py

```diff
@@ -134,7 +135,7 @@ def process(self, data: Shareable, fl_ctx: FLContext):
             ServerCommandKey.SHAREABLE: shareable,
             ServerCommandKey.FL_CONTEXT: copy.deepcopy(get_serializable_data(fl_ctx).props),
         }
-        return data
```

Comment:
What about other CommandProcessor's return result?

Should they also return pickle.dumps(data)?

If so, is there a cleaner way to do this return?